app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,5 @@
+class DashboardController < ApplicationController
+    def index
```

Comment:
The controller action `index` is used for displaying a list of a resource.
Please use a different name for the action/method.

Additional note: I noticed that you are using `index` for the home page in `static pages controller`. Please update this as well. 

reference: https://guides.rubyonrails.org/v2.3/routing.html#restful-routing-the-rails-default